libice will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libice it will fail in the following way:

```
2024/11/12 23:12:07 INFO running step "test/pkgconf"
2024/11/12 23:12:08 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:08 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:08 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:08 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:08 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + basename /home/build/melange-out/libice-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + dev_pkg=libice-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + apk info -L libice-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + grep -q ^Name: usr/lib/pkgconfig/ice.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + basename usr/lib/pkgconfig/ice.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + lib_name=ice uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + echo usr/lib/pkgconfig/ice.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + grep -q '^usr/lib/pkgconfig/ice.pc$\|^usr/share/pkgconfig/ice.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + pkgconf --exists ice uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN + pkgconf --print-errors --exists ice uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN Package xproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN Perhaps you should add the directory containing `xproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 WARN Package 'xproto', required by 'ice', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:08 INFO pod dcc70865b7e940a405e999c3d3decf9b4d8939e512a83912d257eaf6f83d09ae terminated
2024/11/12 23:12:08 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:08 INFO   workspace dir: /home/user/tmp/melange-workspace-546782485
2024/11/12 23:12:08 INFO   guest dir: /home/user/tmp/melange-guest-1496681938
2024/11/12 23:12:08 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libice] Error 1
```

--- a/libice.yaml
+++ b/libice.yaml
@@ -1,7 +1,7 @@
 package:
   name: libice
   version: 1.1.1
-  epoch: 3
+  epoch: 4
   description: X11 Inter-Client Exchange library
   copyright:
     - license: X11
@@ -66,7 +66,11 @@ subpackages:
     dependencies:
       runtime:
         - libice
+        - xorgproto
     description: libice dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

